OpenRA.Platforms.Default/Shader.cs

```diff
@@ -61,29 +53,29 @@ static uint CompileShaderObject(int type, string name)
 				OpenGL.glGetShaderInfoLog(shader, len, out _, log);
 
 				Log.Write("graphics", $"GL Info Log:\n{log}");
-				throw new InvalidProgramException($"Compile error in shader object '{filename}'");
+				throw new InvalidProgramException($"Compile error in shader object.");
```

Comment:
I think preserving the filename to include in this exception will be useful for debugging when editing the shaders.

Reply:
I don't have a reference to the filename here, passing the shader name instead